src/app/helpers/passwordvalidator.ts

```diff
@@ -0,0 +1,21 @@
+import { FormGroup } from '@angular/forms';
+
+// custom validator to check that two fields match
+export function MustMatch(controlName: string, matchingControlName: string) {
```

Comment:
Hey @akram256, please use camelCase for function names, Its conventional for JS.